README.md

```diff
@@ -18,7 +31,17 @@ OPTION template=files/linux.conf
 OPTION devfs_ruleset=${ruleset}
 ```
 
-Where `options/network.makejail` are the options that suit your environment, for example:
+Fetching a base Ubuntu image:
```

Comment:
This applies to what I describe: https://github.com/AppJail-makejails/ubuntu/issues/1#issuecomment-2002536275